tests/examples/deploy.pp

```diff
@@ -0,0 +1,21 @@
+class { 'jboss':
```

Comment:
!fix Remove this example file, and update the working smoke test `tests/deploy.pp` to utilize new `runtime_name` parameter